hpx/include/dist_object.hpp

```diff
@@ -0,0 +1,11 @@
+//  Copyright (c) 2019 Weile Wei
+//
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#if !defined(HPX_DIST_OBJECT_APRIl_12_2019_0412PM)
+#define HPX_DIST_OBJECT_APRIl_12_2019_0412PM
+
+#include <hpx/lcos/dist_object.hpp>
```

Comment:
The name `dist_object` doesn't seem right. I would prefer `distributed_object` - we don't use abbreviations in other classes generally.

Reply:
I changed all the naming from `dist_object` to `distributed_object`.
https://github.com/STEllAR-GROUP/hpx/blob/dist_object_pr/hpx/lcos/distributed_object.hpp#L301
Please review